odilo.py

```diff
@@ -0,0 +1,581 @@
+# coding=utf-8
+import base64
+import datetime
+import isbnlib
+import json
+import logging
+
+from sqlalchemy.orm.session import Session
+
+from httplib import HTTPException
+
+from model import (
+    get_one_or_create,
+    Collection,
+    Contributor,
+    Credential,
+    DataSource,
+    DeliveryMechanism,
+    Edition,
+    ExternalIntegration,
+    Hyperlink,
+    Identifier,
+    Representation,
+    Subject,
+)
+
+from core.util.http import RequestNetworkException
+
+from core.analytics import Analytics
+
+from metadata_layer import (
+    CirculationData,
+    ContributorData,
+    FormatData,
+    IdentifierData,
+    Metadata,
+    LinkData,
+    ReplacementPolicy,
+    SubjectData,
+)
+
+from coverage import (
+    BibliographicCoverageProvider,
+)
+
+from config import (
+    CannotLoadConfiguration,
+)
+
+from testing import DatabaseTest
+
+from util.http import (
+    HTTP,
+    BadResponseException,
+)
+
+
+class OdiloAPI(object):
+    log = logging.getLogger("Odilo API")
+
+    LIBRARY_API_BASE_URL = u"library_api_base_url"
+    BASE_URL = "http://localhost:8080/api/v2"  # Debug value by default
```

Comment:
If you have one live API endpoint, I recommend renaming this to DEFAULT_BASE_URL and putting your live API endpoint here so it can serve as a default. If the API endpoing is different for every library, I recommend getting rid of this altogether.

Reply:
Yes, you are right, we use a different API endpoint URL for each library, so I have deleted BASE_URL.